### ISSUE TITLE
Include Theme Color Palette

### ISSUE THREAD

**Issue by MattRudy** [MattRudy/PowerBI-ThemeTemplates#10]
_Source: Modified from Issue 22 in original repository._

Would be great to be able to change the default colors that appear on the Power BI desktop color palette selection.

**Response by @glitteryqueen:**

You can easily do that either by entering the Theme section on your Power BI or with a JSON like:

```
{ 
    "name": "newcolors, 
    "dataColors": ["#74b9ff", "#0984e3", "#fd79a8", "#fab1a0", "#fdcb6e", "#d63031", "#2d3436", "#55efc4"],
    "background":"#FFFFFF",
    "foreground": "#747480",
    "tableAccent": "#FFFFFF"
}
```
You just set up 8 different colors for your data set, and the background, foreground and table accent.

**Comment by MattRudy** [comment]
Ping @djpirra who created the original issue and @glitteryqueen who provided a great sample snippet - follow this to see the sample integrated soon!

**Comment by MattRudy** [comment]
Resolved by #60 - this is now in the master branch.
